tests/microos/cockpit_check.pm

```diff
@@ -0,0 +1,51 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Basic check for cockpit service
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use transactional;
+use utils qw(systemctl);
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
```

Comment:
What about serial console?

Reply:
I tried, but there is something fishy that doesn't work with transactional update and reboots, that's why I am keeping here root-console.